tests/feature/Issues/Issue1334Test.php

```diff
@@ -13,37 +13,37 @@ function (): void {
             v::key('street', v::stringType()->notEmpty())
                 ->key('region', v::stringType()->notEmpty())
                 ->key('country', v::stringType()->notEmpty())
-                ->keyOptional('other', v::nullOr(v::notEmpty()->stringType()))
+                ->keyOptional('other', v::nullOr(v::notEmpty()->stringType())),
         )->assert(
             [
                 ['region' => 'Oregon', 'country' => 'USA', 'other' => 123],
                 ['street' => '', 'region' => 'Oregon', 'country' => 'USA'],
                 ['street' => 123, 'region' => 'Oregon', 'country' => 'USA'],
-            ]
+            ],
         );
     },
-    'street must be present',
+    '`.0.street` must be present',
```

Comment:
I think that's very cool!! It will make so much easier to identify which message is failing when dealing with nested arrays/objects.